amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/APIField.java

```diff
@@ -24,6 +24,9 @@
 
     @JsonProperty(ActivityEPConstants.FIELD_TYPE)
     private String fieldType;
+    
+    @JsonProperty(ActivityEPConstants.ITEM_TYPE)
+    private String itemType;
```

Comment:
I think now we have a good opportunity to merge fieldType, itemType and elementType in its own class that would define the type of an APIField. Maybe call it APIType? This way we can also better control how it is instantiated, for example that itemType and elementType is always null in all cases except lists.

Also what if we switch now from String to an Enum for itemType and fieldType field? It would help documentation.